addon/components/polaris-range-slider.js

```diff
@@ -174,6 +161,19 @@ export default Component.extend({
    */
   onBlur() {},
 
+  /**
+   * ID for range input
+   *
+   * Defaults to Ember's internal GUID for the component instance
+   *
+   * @property {id}
+   * @type {String}
+   * @public
+   */
+  id: computed(function() {
```

Comment:
Public property shouldn't have moved down

Reply:
@sivakumar-kailasam ^^

This is part of the public interface, and is meant to be overridable - the computed value just provides a default in case nothing is supplied...